src/packages/excalidraw/CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ Please add the latest change on the top under the correct section.
 
 ### Fixes
 
+- Fix raising an error when dragging and dropping some items without files onto the canvas [#4890](https://github.com/excalidraw/excalidraw/pull/4890).
```

Comment:
I think we don't need a changelog for this as the changelog should be reserved for package-affecting changes (/cc @ad1992). But if we did, then we would have to put it under "Unreleased" section.

Reply:
:facepalm: agreed - sorry, I searched for `Fixes` and pasted in the first section I found, my mistake. Will remove in any case as you suggest